content/blog/wingingit/winging-it-25.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Bad UX Design Patterns
+sub: Managing complex navigation with guest Clayton Dewey
+episode: 25
+tags:
+  - OOUX
+author:
+  - stacy
+  - sondra
+date: 2025-10-23
+length: 60 min
+image:
+  src: winging-it/winging-it-25.jpg
+media:
+  - youtube: 2RcF550Pcqc
+summary: |
+  OddBirds, Stacy and Sondra,
+  along with special guest
+  [Clayton Dewey](https://devcollaborative.com/about-us/clayton-dewey)
+  of Dev Collaborative,
+  face off against
+  bad UX design patterns.
+  They get into the weeds
+  trying to address
+  some of the thorniest,
+  perennial UX design questions -
+  managing complex navigation,
+  intuitive user flow,
+  is there such a thing
+  as a good toggle design,
+  and more.
+---
+
+{% callout 'note', false %}
+Check out our [Winging It](/wingingit/)
+conversations about design, frontend,
+and backend development.
+{% endcallout %}
+
+{% import "embed.macros.njk" as embed %}
+{% import "utility.macros.njk" as utility %}
+
+{{ embed.figure(
+  data=media
+) }}
+
+{{ utility.main_action(
+  'Subscribe to Channel »',
+  subscribe_url
+) }}
+
+## What We Cover:
+
+### Complex Navigation
+- OddBird's complex navigation and disperate audiences
+- Defining web "objects" with OOUX philosophy
+- How to prioritize the navigation people need most
+- Are mega menus ever a good idea?
+- Mobile navigation that doesn't bury important content
+- Avoiding nebulous naming
+- OddBird's current navigation solution
+- "Don't make me think."
+- Vitaly’s Idea: Query the user's intent rather than making assumptions
+
+### Intuitive User Flow
+- DevCollab's user flow and many services
+- Content strategy process
+- Card sorting with clients to get user feedback
+
+### UX Design Speed Round
+- Is there ever a case for infinite scroll?
+- Pagination or Load More button?
+- Is there such a thing as a good toggle button design?
+- Click to open or hover?
+- IDs on headers and click to copy
+- Is it a button or a link?
+- Should a link open in the same window or a new window?
+- Down with autoplay!
+- Truncatio...
+- Forms with unclear error messages
+- Bad Apple remote interface design
+
+## Links:
+
+- [oddbird.dev](http://oddbird.dev/)
```

Comment:
Have we settled on oddbird.dev? We're not using it consistently, and I'm not sure some email aliases e.g. `birds@oddbird.dev` have been tested.
```suggestion
- [oddbird.net](/)
```